Modules/_lsprof.c

```diff
@@ -479,27 +489,23 @@ static PyStructSequence_Field profiler_subentry_fields[] = {
 
 static PyStructSequence_Desc profiler_entry_desc = {
     "_lsprof.profiler_entry", /* name */
```

Comment:
Rather than comment to give the member names, I would prefer to use C99 style:

```
static PyStructSequence_Desc profiler_entry_desc = {
    .name = "_lsprof.profiler_entry",
    .doc = "",
    .fields = profiler_entry_fields,
    .n_in_sequence = 6,
};
```

Can you please use this style for profiler_entry_desc and profiler_subentry_desc?